src/panels.cpp

```diff
@@ -1756,6 +1756,135 @@ static void draw_compass( avatar &, const catacurses::window &w )
     wnoutrefresh( w );
 }
 
+// Forward declarations
+std::string direction_to_enemy_improved( const tripoint &enemy_pos, const tripoint &player_pos );
+void check( const char *msg, std::function<std::string( const tripoint &, const tripoint & )> fn );
```

Comment:
```suggestion
void check( const char *msg, std::function<auto ( const tripoint &, const tripoint & ) -> std::string> fn );
```
also the name `check` is vague, and only used in `main` function, is that intentional it's not used in other part of the code?

Reply:
I don't know, lol, I just take what is suggested to me in the improved thing and work with that